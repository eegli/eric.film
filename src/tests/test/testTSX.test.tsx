import { render } from '@/src/tests/testUtils';

const Page: React.FC = () => <div>Test</div>;

describe('Test - TSX', () => {
  it('renders test TSX component', () => {
    const { asFragment } = render(<Page />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
