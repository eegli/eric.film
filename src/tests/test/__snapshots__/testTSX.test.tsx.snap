// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test - TSX renders test TSX component 1`] = `
<DocumentFragment>
  <div>
    Test
  </div>
</DocumentFragment>
`;
